### PR TITLE
Sync conda runtime dependencies with pyproject.toml

### LINKED ISSUE #91: `setup.cfg` was updated
The file `setup.cfg` was modified in `napari/napari`. This might mean the conda-recipe in
`napari/packaging` needs adjusting! Please check!

Calculated diff:

```diff
diff --git a/setup.cfg b/setup.cfg
index 98a06c61..ab5db46c 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -72,7 +72,6 @@ install_requires =
     scikit-image[data]>=0.19.1 # just `pooch`, but needed by `builtins` to provide all scikit-image.data samples
     scipy>=1.4.1 ; python_version < '3.9'
     scipy>=1.5.4 ; python_version >= '3.9'
-    sphinx>=4.3.0,<5  # numpydoc dependency. sphinx>=5 breaks the docs build; see https://github.com/napari/napari/pull/4915
     superqt>=0.5.0
     tifffile>=2020.2.16
     toolz>=0.10.0
```

cc @jaimergp

### PR DESCRIPTION
**Pull request by jaimergp**
Closes #91

### CODE REVIEW

**Comment by jaimergp** [comment]
@Czaki do you need to hold this while we wait for 0.4.19 too?

**Comment by Czaki** [comment]
I think yes.

**Review by goanpeca — APPROVED** [review]

File: conda-recipe/meta.yaml [hunk 1]
```diff
@@ -80,7 +85,7 @@ outputs:
         - pyside2 >=5.14.2,!=5.15.0
         - pyqt >=5.12.3,!=5.15.0
         - pyside6 <6.3.2
-
+        - numba >=0.57.1  # https://github.com/napari/napari/pull/6467#issuecomment-1819038165
```
Comment (jaimergp) [inline conda-recipe/meta.yaml:88]:
```suggestion
        # Nice for performance, but optional. Inclusion under discussion.
        # - py-triangle
        # - numba >=0.57.1  # https://github.com/napari/napari/pull/6467#issuecomment-1819038165
```

File: conda-recipe/meta.yaml [hunk 2]
```diff
@@ -92,6 +99,7 @@ outputs:
       #   - xarray
       #   - pandas
       #   - pytest-timeout
+      #   - numba >=0.57.1  # https://github.com/napari/napari/pull/6467#issuecomment-1819038165
```
Comment (jaimergp) [inline conda-recipe/meta.yaml:102]:
```suggestion
      #   - py-triangle
      #   - numba >=0.57.1  # https://github.com/napari/napari/pull/6467#issuecomment-1819038165
```

**Pull request merged**